app/src/main/java/ar/com/wolox/android/example/model/User.java

```diff
@@ -0,0 +1,61 @@
+package ar.com.wolox.android.example.model;
+
+import com.google.gson.annotations.SerializedName;
+import java.io.Serializable;
+
+/**
+ *
+ */
+public class User implements Serializable {
+
+   @SerializedName("id") private int id;
+   @SerializedName("username") private String username;
+   @SerializedName("email") private String email;
+   @SerializedName("password") private String password;
+   @SerializedName("picture") private String picture;
+   @SerializedName("cover") private String cover;
+   @SerializedName("description") private String description;
+   @SerializedName("location") private String location;
+   @SerializedName("name") private String name;
+   @SerializedName("phone") private String phone;
+
+    public int getId() {
```

Comment:
We have to add Nullabilities for this methods. Try to use @Nullable or @Nonnull

Reply:
✔ **Done**

But assuming email, username and password can't be null and the other fields can be null. I don't know if that assumption is true...